src/shared/wizards/multiStepWizard.ts

```diff
@@ -49,7 +49,7 @@ export class WorkspaceFolderQuickPickItem implements FolderQuickPickItem {
 
 export class WizardContext {
     public readonly showOpenDialog = vscode.window.showOpenDialog
-    public get workspaceFolders(): vscode.WorkspaceFolder[] | undefined {
+    public get workspaceFolders(): ReadonlyArray<vscode.WorkspaceFolder> | undefined {
```

Comment:
Not necessarily related to this change, but is there a benefit to using this in place of just calling vscode directly?

Reply:
the wizard is pretty overwrought, but refactoring it is out of scope here.